libretro/libretro.c

```diff
@@ -719,7 +719,7 @@ void retro_init(void)
         initializing = true;
 
         retro_thread = co_active();
-        game_thread = co_create(65536 * sizeof(void*) * 16, EmuThreadFunction);
+        game_thread = co_create(65536 * sizeof(void*) * 16, (void(*)(void))EmuThreadFunction);
```

Comment:
I think https://github.com/libretro/mupen64plus-libretro-nx/pull/526/commits/26dfd670ffdd5ed6a03e6704dc73f82c13d81dd9 is a bit better than this workaround.